omnibus/files/private-chef-cookbooks/private-chef/templates/default/nginx/scripts/check_token_using_endpoint.lua.erb

```diff
@@ -0,0 +1,24 @@
+ngx.req.read_body()
+local res
+
+if ngx.var.request_method == "POST" then
+  meth = ngx.HTTP_POST
+else
+  meth = ngx.HTTP_GET
+end
+
+res = ngx.location.capture("/organizations/" .. ngx.var.request_org .. "/validate" .. ngx.var.request_uri,
+    { method = meth,
+      always_forward_body = true,
+      copy_all_vars = true,
```

Comment:
From my understanding, if we have `copy_all_vars = true`, the next line is superfluous, isn't it?

> `vars` take a Lua table which holds the values to set the specified Nginx variables in the subrequest as this option's value.
> `copy_all_vars` specify whether to copy over all the Nginx variable values of the current request to the subrequest in question. modifications of the nginx variables in the subrequest will not affect the current (parent) request. ([src](https://github.com/openresty/lua-nginx-module#ngxlocationcapture))

Since `upstream` is in `vars`, I don't think we need this line. (But see my other comment, maybe we don't even need upstream.)

We also don't need `request_org` here, since it's passed to the validate endpoint as part of the scoped URL. (This was a relict from trying to get this work in the mocked script/config used with Test::Nginx.)